repository/Seaside-Core.package/WAServerManager.class/class/default.st

```diff
@@ -1,4 +1,6 @@
 accessing
 default
-	default contents ifNil: [ default contents: self new ].
-	^ default contents
\ No newline at end of file
+
+	self initializeDefaultValueHolder.
```

Comment:
Perhaps renaming the method `initializeDefaultValueHolder` to `ensureDefaultValueHolder` makes it clear that `Default` is not reinitialized when invoking this method? 
I was specifically triggered by seeing this method call here while the value of `Default` should remain the same `WAValueHolder` instance (or `TransientValue` in GemStone)

Reply:
Hey @eMaringolo. What do you think about my suggestion?

Reply:
Sorry, I missed this. I'm okay with your suggestion, I think it is more intention revealing that the one with the `initialize` prefix.

Reply:
I will merge and rename